components/dashboard/src/projects/Project.tsx

```diff
@@ -184,6 +188,20 @@ export default function () {
         return date ? dayjs(date).fromNow() : "";
     };
 
+    const resumePrebuilds = async () => {
+        if (!project) {
+            return;
+        }
+        try {
+            const response = await getGitpodService().server.triggerPrebuild(project.id, null);
```

Comment:
Looks like you're missing a `setIsResuming(true)` somewhere. Maybe add it here, before the trigger?

```suggestion
            setIsResuming(true);
            const response = await getGitpodService().server.triggerPrebuild(project.id, null);
```

Reply:
Thanks for catching this! Indeed, 'failed to push harder.